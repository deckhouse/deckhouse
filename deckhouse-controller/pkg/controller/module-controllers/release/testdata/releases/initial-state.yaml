---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: commander-1.0.1
  labels:
    module: commander
    modules.deckhouse.io/update-policy: commander
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
spec:
  moduleName: commander
  version: 1.0.1
  weight: 900
status:
  phase: Superseded
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: commander-1.0.2
  labels:
    module: commander
    modules.deckhouse.io/update-policy: commander
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
spec:
  moduleName: commander
  version: 1.0.2
  weight: 900
status:
  phase: Deployed
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: commander-1.0.3
  labels:
    module: commander
    modules.deckhouse.io/update-policy: commander
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
spec:
  moduleName: commander
  version: 1.0.3
  weight: 900
status:
  phase: Pending
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: parca-1.2.1
  labels:
    module: parca
    modules.deckhouse.io/update-policy: parca
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
spec:
  moduleName: parca
  version: 1.2.1
  weight: 900
status:
  phase: Deployed
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: parca-1.2.2
  labels:
    module: parca
    modules.deckhouse.io/update-policy: parca
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
spec:
  moduleName: parca
  version: 1.2.2
  weight: 900
status:
  phase: Pending
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: c12a99d96e9ec5e0b07d24cc9615835f
  finalizers:
    - modules.deckhouse.io/release-exists
  name: test
---
apiVersion: deckhouse.io/v1alpha2
kind: ModuleUpdatePolicy
metadata:
  name: parca
spec:
  releaseChannel: Alpha
  update:
    mode: Auto
---
apiVersion: deckhouse.io/v1alpha2
kind: ModuleUpdatePolicy
metadata:
  name: commander
spec:
  releaseChannel: Alpha
  update:
    mode: Auto
