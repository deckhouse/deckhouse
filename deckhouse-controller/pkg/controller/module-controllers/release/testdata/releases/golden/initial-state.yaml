---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleSource
metadata:
  annotations:
    modules.deckhouse.io/registry-spec-checksum: c12a99d96e9ec5e0b07d24cc9615835f
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/release-exists
  name: test
  resourceVersion: "999"
spec:
  registry:
    ca: ""
    dockerCfg: ""
    repo: ""
status:
  message: ""
  modules: null
  modulesCount: 0
  phase: ""
  syncTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: commander
    modules.deckhouse.io/update-policy: commander
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
  name: commander-1.0.1
  resourceVersion: "999"
spec:
  moduleName: commander
  version: 1.0.1
  weight: 900
status:
  approved: false
  message: ""
  phase: Superseded
  pullDuration: 0s
  size: 0
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: commander
    modules.deckhouse.io/update-policy: commander
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
  name: commander-1.0.2
  resourceVersion: "1000"
spec:
  moduleName: commander
  version: 1.0.2
  weight: 900
status:
  approved: false
  message: ""
  phase: Superseded
  pullDuration: 0s
  size: 0
  transitionTime: "2019-10-17T15:33:00Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    modules.deckhouse.io/isUpdating: "true"
    modules.deckhouse.io/notified: "false"
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/exist-on-fs
  labels:
    module: commander
    modules.deckhouse.io/update-policy: commander
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
    status: deployed
  name: commander-1.0.3
  resourceVersion: "1001"
spec:
  moduleName: commander
  version: 1.0.3
  weight: 900
status:
  approved: false
  message: ""
  phase: Deployed
  pullDuration: 9m15s
  size: 0
  transitionTime: null
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  creationTimestamp: null
  labels:
    module: parca
    modules.deckhouse.io/update-policy: parca
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
  name: parca-1.2.1
  resourceVersion: "1000"
spec:
  moduleName: parca
  version: 1.2.1
  weight: 900
status:
  approved: false
  message: ""
  phase: Superseded
  pullDuration: 0s
  size: 0
  transitionTime: "2019-10-17T15:33:00Z"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  annotations:
    modules.deckhouse.io/isUpdating: "true"
    modules.deckhouse.io/notified: "false"
  creationTimestamp: null
  finalizers:
  - modules.deckhouse.io/exist-on-fs
  labels:
    module: parca
    modules.deckhouse.io/update-policy: parca
    release-checksum: fd59af7328ed891b53cc999f242b0866
    source: test
    status: deployed
  name: parca-1.2.2
  resourceVersion: "1001"
spec:
  moduleName: parca
  version: 1.2.2
  weight: 900
status:
  approved: false
  message: ""
  phase: Deployed
  pullDuration: 9m15s
  size: 0
  transitionTime: null
